packages/recma-jsx/package.json

```diff
@@ -12,6 +12,12 @@
     "recma-stringify": "^1.0.0",
     "unified": "^11.0.0"
   },
+  "devDependencies": {
+    "acorn": "^8.0.0"
+  },
```

Comment:
Just the peer dependencies should be sufficient (for npm, which we use here).

```suggestion
```

Reply:
I do agree.